components/datePicker/datePicker.module.scss

```diff
@@ -1,7 +1,7 @@
 .cont input {
   background-color: var(--color-dark-700);
   height: 36px;
```

Comment:
Date pickers inner `input` tag should be accessed by the class name, which is already provided. It is `rmdp-input`. Avoid using bare tag-names in CSS selectors. In order to retain provided class-names in global scope during CSS-modules compilation expose `:global` selector.

Final selector might look like this:
```scc
.datePickerContainer {
  :global {
    .rmdp-input {
      // your code here...
    }
  }
}
```

Reply:
done